packages/flutter/test/material/app_bar_test.dart

```diff
@@ -217,7 +217,7 @@ void main() {
     final Finder title = find.byKey(titleKey);
     expect(tester.getTopLeft(title).dx, 72.0);
     // The toolbar's contents are padded on the right by 4.0
-    expect(tester.getSize(title).width, equals(800.0 - 72.0 - 4.0));
+    expect(tester.getSize(title).width, equals(800.0 - 4.0 - 16.0 - 56.0 - 16.0));
```

Comment:
Ah, cool.

If you're updating the actual layout algorithm as well, please also regenerate https://github.com/flutter/assets-for-api-docs/blob/master/material/app_bar.png (there's a dart file in that directory that you can drop into a `flutter create` app and run to generate the image, you'll need imagemagick and optipng).